packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx

```diff
@@ -6,15 +6,15 @@ import { AppHeaderClickHandler, AppHeaderLinkItem } from '../types';
 export type AppHeaderLinkProps = {
   action: AppHeaderClickHandler;
   item: AppHeaderLinkItem;
-  isMobile?: boolean;
+  showIcons?: boolean;
```

Comment:
```suggestion
  showIcon?: boolean;
```

Super non blocking nit since a link would only have 1 icon